assets/js/analyzer.js

```diff
@@ -143,6 +143,10 @@ function populateSecondaryAnalyzerList() {
 }
 
 function analyze() {
+    if($('#primaryAnalyzerMode').val() === null) {
```

Comment:
@sushain97  Even I was confused. I did a git diff on my local machine between this branch and master branch and the only changes are with respect to the functionality of button. Could you help me understand why it might be happening ? I am updating this PR to check if this is resolved. Thanks!